walletlibrary/src/main/java/com/microsoft/walletlibrary/util/Constants.kt

```diff
@@ -4,4 +4,9 @@ object Constants {
     const val OPENID_SCHEME = "openid-vc"
     const val PURE_ISSUANCE_FLOW_VALUE = "create"
     const val IDTOKENHINT_CONFIGURATION = "https://self-issued.me"
+
+    // Feature flags for Access Token flow preview feature.
+    const val FEATURE_FLAG_OPENID4VCI_ACCESS_TOKEN = "OpenID4VCIAccessToken"
```

Comment:
One more thing, it might be better to have these Constants in a Separate Public Feature Flag supported class so that they are easy to inject into the builder instead of having the developer use that specific string, they can just use a Public Constant.

Reply:
It makes sense in usability for developers. I was going for grouping Constants together in one place. I will make the change.